src/base/bittorrent/torrentimpl.h

```diff
@@ -319,6 +322,7 @@ namespace BitTorrent
         bool m_hasFirstLastPiecePriority = false;
         bool m_useAutoTMM;
         bool m_isStopped;
+        bool m_hasWorkingTracker = true;
```

Comment:
I want this to be initialized as true because on startup trackers might be still updating or not contacted state and I do not want such torrents to show up in this filter.